DEVELOPMENT.md

```diff
@@ -143,3 +150,33 @@ And finally, run `yarn run db:update` to update the datastore schema.
 ```
 yarn run db:update
 ```
+
+Now, start the app back up, and you should start seeing new data coming from Notehub to your new local Postgres instance.
+
+```
+yarn run dev
+```
+
+### Check the Postgres DB is receiving data
+
+If you'd like to connect to your locally running Postgres instance to ensure new Notehub events are being added, you can use the browser or local-based [Prisma Studio](https://www.prisma.io/studio) to easily check.
+
+In a new terminal window run the following command:
+
+```
+npx prisma studio
+```
+
+This will open up a new browser window at http://localhost:5555 where you can see your Prisma DB, its tables, and any data that currently resides therein, whether it's preseeded or has been routed in by Notehub.
+
+And just like any other database GUI, you can click into models to view data, manipulate data, filter, query, etc., etc.
+
+### Troubleshooting Sparrow getting data from Postgres
+
+There's a number of gotchas that could be the reason your Notehub data's not making it to Postgres and your Sparrow app - try checking the following things:
+
+- Is Docker running the local Postgres instance on your machine?
+  - Currently there's no error message thrown if the Postgres Docker container's not running
+- Does your Ngrok endpoint match what's in Notehub and have the suffix `/api/datastore/ingest`?
+  - Be aware, every time the Ngrok connection is shut down and restarted, it will be started up with a brand new URL, so you'll need to update the route according in Notehub to ensure data keeps flowing to it
```

Comment:
kill “according”

Reply:
Thanks - I think I had a brain malfunction when that word was somehow added to the sentence 😅